src/components/TutorialPage/index.tsx

```diff
@@ -283,17 +283,20 @@ const TutorialPage = (): JSX.Element => {
                     <h1 id="browser-support">Browser support</h1>
                     <ul>
                         <li>
-                            Currently, Simularium supports Firefox, Chrome, and
-                            Edge. Some features may not work on other browsers.
+                            Currently, Simularium supports Firefox, Safari,
+                            Chrome, and Edge. Some features may not work on
+                            other browsers.
                         </li>
                         <li>
-                            If using Safari on a Mac, please enable WebGL 2.0 by
-                            choosing Develop &gt; Experimental Features and
-                            enabling &quot;WebGL 2.0&quot; (If you do not have a
-                            Develop menu in your menu bar, please first choose
+                            If using Safari on a Mac and it is not working,
+                            update your browser to the most recent version. If
+                            you are using an older version, please enable WebGL
+                            2.0 by choosing Develop &gt; Experimental Features
+                            and enabling &quot;WebGL 2.0&quot; (If you do not
+                            have a Develop menu in your menu bar, first choose
                             Safari &gt; Preferences &gt; Advanced and enable
                             &quot;Show Develop menu in menu bar&quot;.) Then
-                            please reload the viewer.
+                            reload the viewer.
```

Comment:
There were too many 'pleases' in this for me when I read it over, but I can add it back.

Reply:
I wouldn't say "please" at all tbh. The fewer words the better.

Reply:
I agree, please remove all the instances of the word please. Pretty please with sugar on top.

I wonder if we can link people to Apple to get full instructions on how to enable WebGL2 on older safari.